cluster/src/Cardano/Cluster.hs

```diff
@@ -72,18 +70,9 @@ startCluster prefix nodes = do
             let (artifacts, nodeEnv) = prepareEnvironment node nodes stateDir env
             let (genesis, topology, logger, tls) = artifacts
 
-            case nodeType of
-                NodeCore -> do
-                    void (init genesis >> init topology >> init logger)
-                    yield (RunningCoreNode nodeId nodeEnv)
-
-                NodeRelay -> do
-                    void (init topology >> init logger)
-                    yield (RunningRelayNode nodeId nodeEnv)
-
-                NodeEdge -> do
-                    manager  <- init topology >> init logger >> init tls
-                    yield (RunningEdgeNode nodeId nodeEnv manager)
```

Comment:
Simpler now. Basically, only core nodes have an extra step of generating the genesis keys. The rest is now identical :tada: